oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,138 @@
+authoring process. The Publisher tool includes an inflexible workflow engine for the process of provisioning
+a course. Currently, this course publish cycle can take up to fifteen days in order to provision 
+a new course. 
+
+We also run into data synchronization issues between the separate services since there are multiple sources of truth and
+multiple writers of the relevant metadata.
+
+Decisions
+---------
+
+* We will no longer embed workflow into the core system. Rather, we will allow for a self-service workflow tool that will reduce the provisioning time.
+
+* Conceptually, we will separate the notion of Course within the edX system in the following ways:
+
+  * **Course Learning Content**, a course-team created package of learning material, with its weekly and sub-weekly structure
+    (i.e., Sections, Subsections, Units, etc) and learning blocks (i.e., leaf xBlocks), tied to specific dates of offering.
+     
+    * This is what we call **(Studio) Course Run**, for the time being.
+    * It is bound to the organization that authors the content.
+    * Note: In the future, this concept may be further separated when we support content reuse across "Course Runs".
+      At that time, we may distinguish between:
+
+      * **Reusable Learning Content** (not tied down to specific dates nor structure) and
+      * **Packaged Learning Content** (an "instantiation" of a "Reusable Learning Content" with structure, dates, and 
+        configuration).
+
+  * **Discoverable Course**, a learner-facing representation of a course that entices enrollment with marketing information
+    such as the course's price and credit/degree outcomes.
+
+    * This is what we call **Catalog Course**, for the time being.
+    * It is bound to a specific marketing site.
+
+  * **Learner Consumable Course**, a learner-engaging course, with read-optimized representation of the learning content. 
+
+    * This is what we call **(LMS) Course Run**.
+    * It is bound to a specific learner.
+
+* Given the above distinctions, each `edX subdomain`_ will manage its own concept. Hence, the authoring logic
+  implementation for each concept will also remain separate. This prevents us from needing to maintain these concepts across
+  multiple domains: keep the authoring of Discoverable-Course (**Catalog Course**) within Discovery; keep the authoring of 
+  Learning-Content-Course (**Course Run**) within Content Authoring. For example, concepts of price, marketing-publish-date,
+  marketing sales, discounts, etc do not need to bleed into the Content Authoring service.
+
+* However, from the course author's perspective, we will provide a unified user experience by having the current Studio tool
+  be the portal to authoring both Course Run metadata and Catalog Course metadata. So even though authoring of Catalog Course
+  metadata will live in the Discovery subdomain, there will be a seamless transition between the front ends of Course Run
```

Comment:
does this mean for editing catalog data, people will log into the https://prod-edx-discovery.edx.org/ subdomain?  I thought this OEP was about unifying the edit into a single tool...